bin/node/runtime/src/lib.rs

```diff
@@ -642,7 +642,10 @@ impl Get<Option<(usize, ExtendedBalance)>> for OffchainRandomBalancing {
 }
 
 impl onchain::Config for Runtime {
```

Comment:
for readability purposes, let's implement `onchain::config` for some new type that describes it a bit: `impl<T: Config> onchain::Config for OnchainPhragmen`, and they use it below.

Reply:
ahh, I see that I originally bounded `onchain::Config` itself to be `frame_system::Config` to discount myself needing to declare `AccountId` and `BlockNumber`. Now's the time to revert that little bit as well :) I think letting `onchain::Config` no longer being bounded by `system::Config` is a good idea here.

Reply:
I have refactored that bit, it's quite a bit of changes.